src/Discord.Net/DiscordConfig.cs

```diff
@@ -5,7 +5,7 @@ namespace Discord
     public class DiscordConfig
     {
         public const int APIVersion = 6;        
-        public static string Version { get; } = typeof(DiscordRestConfig).GetTypeInfo().Assembly?.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion ?? "Unknown";
+        public static string Version { get; } = typeof(DiscordRestConfig).GetTypeInfo().Assembly?.GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion ?? "Unknown";
```

Comment:
When is InformationVersion not available? Rather than returning Unknown, it should fall back to the old way of getting version: Assembly.GetName().Version.ToString(3)